views/scss/qti/_choice.scss

```diff
@@ -44,6 +44,7 @@
             left: 50%;
             transform: translateX(-50%);
             bottom: calc(100% + 20px);
+            white-space: nowrap;
```

Comment:
As I see this is the only change. Is the correct? https://github.com/oat-sa/extension-tao-itemqti/pull/2296/files?diff=split&w=1#r1047256383

Reply:
Yes, right